### ISSUE TITLE
Task watchdog got triggered for adv_post_task when trying to send statistics when there is no network connection

### ISSUE THREAD

**Issue by TheSomeMan** [ruuvi/ruuvi.gateway_esp.c#401]
```
I (3231) ruuvi_gateway: [main] Main loop started
I (11621) ADV_POST_TASK: [adv_post_task] Advertisements in table: 1
I (11622) ADV_POST_TASK: [adv_post_task] i: 0, tag: F4:C6:46:2C:3E:B4, rssi: -48, timestamp: 1635420253:
I (11627) ADV_POST_TASK: [adv_post_task] 0000: 02 01 06 1B FF 99 04 05 14 26 1F 98 C5 60 00 24 | .........&...`.$
I (11637) ADV_POST_TASK: [adv_post_task] 0010: 00 0C 03 F8 AA 16 47 C2 25 F4 C6 46 2C 3E B4    | ......G.%..F,>.
W (11647) ADV_POST_TASK: [adv_post_task] Can't send, no network connection
I (11657) http: [adv_post_task] HTTP POST to URL=https://network.ruuvi.com/status, DATA:
{
        "DEVICE_ADDR":  "C8:25:2D:8E:9C:2C",
        "ESP_FW":       "v1.8.3-9-g9a5b68d-dirty",
        "NRF_FW":       "v0.7.2",
        "UPTIME":       "8",
        "NONCE":        "3704167872",
        "CONNECTION":   "ETHERNET",
        "NUM_CONN_LOST":        "0",
        "SENSORS_SEEN": "1",
        "ACTIVE_SENSORS":       [{
                        "MAC":  "F4:C6:46:2C:3E:B4",
                        "COUNTER":      "7"
                }],
        "INACTIVE_SENSORS":     []
}
I (13189) ruuvi_gateway: [main] free heap: 121152
E (16055) task_wdt: Task watchdog got triggered. The following tasks did not reset the watchdog in time:
E (16055) task_wdt:  - adv_post_task (CPU 0/1)
E (16055) task_wdt: Tasks currently running:
E (16055) task_wdt: CPU 0: IDLE0
Guru Meditation Error: Core  0 panic'ed (Interrupt wdt timeout on CPU0)
Core 0 register dump:
PC      : 0x4008a76c  PS      : 0x00060434  A0      : 0x8008b99c  A1      : 0x3ffb0540  
A2      : 0x3ffe2524  A3      : 0x3ffc1200  A4      : 0x3ffb0014  A5      : 0x3ffd1280  
A6      : 0x00000000  A7      : 0x00000001  A8      : 0x3ffc1200  A9      : 0x3ffc1200  
A10     : 0x00000019  A11     : 0x00000019  A12     : 0x3ffd1190  A13     : 0x00000000  
A14     : 0x00000000  A15     : 0x3ffe24e8  SAR     : 0x00000004  EXCCAUSE: 0x00000005  
EXCVADDR: 0x00000000  LBEG    : 0x4000c2e0  LEND    : 0x4000c2f6  LCOUNT  : 0xffffffff  
Core 0 was running in ISR context:
EPC1    : 0x40148cc4  EPC2    : 0x00000000  EPC3    : 0x00000000  EPC4    : 0x4008a76c

ELF file SHA256: 167fca3681f98705

Backtrace: 0x4008a769:0x3ffb0540 0x4008b999:0x3ffb0560 0x40089bb3:0x3ffb0580 0x4012d049:0x3ffb05c0 0x40153c4e:0x3ffb05e0 0x401491de:0x3ffb0600 0x400819d2:0x3ffb0620 0x400d9747:0x3ffb0640 0x400d4dfc:0x3ffb0660 0x400827d2:0x3ffb0680 0x40190cf3:0x3ffc10f0 0x400d56fe:0x3ffc1110 0x4008b205:0x3ffc1130 0x4008a7d1:0x3ffc1150

Rebooting...
ets Jul 29 2019 12:21:46
```

**Comment by markoaamunkajo** [comment]
[Uploading loki.txt…]()
Task watchdog got triggered. Spotted in logs 1.9.0

**Comment by TheSomeMan** [comment]
@markoaamunkajo is seems that the link to loki.txt is broken

**Comment by TheSomeMan** [comment]
It's another problem, see #424, fixed in v1.9.1

**Comment by markoaamunkajo** [comment]
No more watchdog reports in log, moving to done
